platform/darwin/src/MGLVectorTileSource.mm

```diff
@@ -109,7 +109,9 @@ + (NSString *)preferredMapboxStreetsLanguage {
 + (NSString *)preferredMapboxStreetsLanguageForPreferences:(NSArray<NSString *> *)preferencesArray {
     BOOL acceptsEnglish = [preferencesArray filteredArrayUsingPredicate:
                            [NSPredicate predicateWithBlock:^BOOL(NSString * _Nullable language, NSDictionary<NSString *,id> * _Nullable bindings) {
-        return [[NSLocale localeWithLocaleIdentifier:language].languageCode isEqualToString:@"en"];
+
+        NSString *languageCode = [[NSLocale localeWithLocaleIdentifier:language] objectForKey:NSLocaleLanguageCode];
```

Comment:
Is it possible to wrap in a `if #available` statement so when we remove iOS 9 support we also remove this code and keep the latest API?

Reply:
Yep we can do that - I'll modify the previous case of this too